app/data/helpers/generators.js

```diff
@@ -0,0 +1,72 @@
+
+
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+exports.firstName = (sex) => {
```

Comment:
Can you sort the list for ease on the eyes?

Reply:
okay